MekHQ/src/mekhq/campaign/CampaignOptions.java

```diff
@@ -49,6 +38,12 @@
 import mekhq.campaign.rating.UnitRatingMethod;
 import mekhq.service.mrms.MRMSOption;
 import mekhq.utilities.MHQXMLUtility;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+import java.io.PrintWriter;
+import java.util.*;
+import java.util.Map.Entry;
 
```

Comment:
This is going to be a nitpick related to the style guide and the recently added EditorConfig MIGHT trigger for your IDEs.

Imports alphabetized, statics at top, and the wildcard filter set to 10 before triggering.

Reply:
Where can I configure that? Or will I need to be doing that by hand?

Reply:
If using IntelliJ, https://www.jetbrains.com/help/idea/editorconfig.html should help enable it IF you have master merged in with the config file.

The config file probably still needs some tweaks to get it to match the style where it can and to work with the various IDEs but it should be a good base for now.